src/components/Card/Card.js

```diff
@@ -31,22 +65,27 @@ const WrapperList = styled.li`
   box-shadow: 0 8px 16px #0006;
   display: grid;
   max-width: 100%;
-
   font-family: 'Roboto';
   list-style-type: none;
-  font-size: 20px;
+  font-size: 16px;
```

Comment:
You could think about using rem instead of px (16px should be 1 rem unless you have a zoom in base styling)